### ISSUE TITLE
Question about image with alt="" and aria-label="foo"

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria#902]
We got a question on WCAG that may be more appropriate over here. Thoughts?

https://github.com/w3c/wcag/issues/521

**Comment by DavidMacDonald** [comment]
PS my guess is that the alt should be ignored by AT and the aria-label read. And WCAG conformance should be determined based on that algorithm.

**Comment by jnurthen** [comment]
Sounds like accname to me. If you agree can you log it against the correct spec.

**Comment by DavidMacDonald** [comment]
OK
